### DISCUSSION TITLE
Can't delete a project (opportunity-level)

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#397] [General] [answered]
I [created a test-project](https://hub.alkem.io/digicampus/challenges/virtassist/opportunities/voiceid) just to see how it works. I see there is no way of deleting it. It is not a big deal for now since the project-functionality is something for the future.

**Comment by techsmyth** [comment, marked as answer]
Hi Flori, indeed the client does not currently expose the ability to delete Projects. Added a card for this up under https://github.com/alkem-io/client-web/issues/925

Note: the deletion of Projects is exposed on the api:

![image](https://user-images.githubusercontent.com/30729240/128331093-03318ac8-8b1a-4b78-9207-22be56bce23d.png)
